scripts/hal_alignment/hal_to_maf.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Extract a MAF alignment from a HAL file.
+
+Examples::
+    # Extract a MAF alignment with chromosome 19 of GRCh38 as reference.
+    python hal_to_maf.py --ref-genome GRCh38 --ref-sequence chr19 \
+      --keep-genomes-file genome_list.txt input.hal output.maf
+
+"""
+
+from argparse import ArgumentParser
+import subprocess
+
+from Bio.AlignIO.MafIO import MafIterator, MafWriter
+
+
+if __name__ == '__main__':
+
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument('hal_file', help="Input HAL file.")
+    parser.add_argument('maf_file', help="Output MAF file.")
+
+    parser.add_argument('--ref-genome', metavar='STR', required=True,
+                        help="Reference genome for output MAF file.")
+    parser.add_argument('--ref-sequence', metavar='STR', required=True,
+                        help="Output sequence within reference genome.")
```

Comment:
I wouldn't consider this a renaming, just a change to meet standard Python argument naming: the arguments are named the same and have the same type and purpose.

Reply:
OK, I will follow the consensus and change `--ref-genome` and `--ref-sequence`
to `--refGenome` and `--refSequence`, respectively.

I've used the `metavar` arguments to prettify the command-line help output. Here are how these arguments would look with `metavar` set...
```
--refGenome STR       Reference genome for output MAF file.
--refSequence STR     Output sequence within reference genome.
```

...and without `metavar` set:
```
--refGenome REF_GENOME
                      Reference genome for output MAF file.
--refSequence REF_SEQUENCE
                      Output sequence within reference genome.
```

Reply:
Suggestion applied in [commit d4201b6](https://github.com/Ensembl/ensembl-compara/pull/374/commits/d4201b69cf288c4dd3e5bae960cf456bade650cd).

Reply:
I have checked the convention, and in this case (and the following one), because the arguments are converted to variables in the code, the convention states that they should be lowercase, with words separated by underscores, i.e. `--ref_genome` and `--ref_sequence`. This doesn't change anything for the underlying script, as it is very similar and has the same function.